frontend/integration-tests/views/monitoring.view.ts

```diff
@@ -32,3 +33,66 @@ export const modalConfirmButton = $('#confirm-action');
 export const alertManagerYamlForm = $('.co-alert-manager-yaml__form');
 export const successAlert = $('.pf-m-success');
 export const helpText = $('.co-help-text');
+
+// Configuration Overview
+export const alertRoutingHeader = $$('h2').first();
```

Comment:
We should use `data-test-id` here

Reply:
I tried adding `<SectionHeading text="Alert Routing" data-test-id="alert-routing">` but `data-test-id` doesn't get passed down to underlying HTML.

Reply:
You can add it inside `SectionHeading`. Something like

```tsx
export const SectionHeading: React.SFC<SectionHeadingProps> = ({ text, children, style }) => (
  <h2 className="co-section-heading" style={style} data-test-section-heading={text}>
    {text}
    {children}
  </h2>
);
```

Then test for the presence of `$('[data-test-section-heading="Alert Routing"]')`

Reply:
thanks, fixed.  What's the issue with using `id`?

Reply:
* `id` needs to be unqiue on the page
* `id` is used for other things like `<label for="my-id">`. `data-test-*` makes it clear the attribute is only there for testing.
* certain characters are not allowed in `id`
* `data-test-*` attributes can have values other than an identifier